subsys/net/ip/Kconfig

```diff
@@ -349,6 +349,42 @@ config NET_BUF_USER_DATA_SIZE
 	  Example: For Bluetooth, the user_data shall be at least 4 bytes as
 	  that is used for identifying the type of data they are carrying.
 
+choice
+	prompt "Default Network Interface"
+	default NET_DEFAULT_IF_NONE
+	help
+	  If system has multiple interfaces enabled, then user shall be able
+	  to choose default interface. Otherwise first interface will be the
+	  default interface.
+
+config NET_DEFAULT_IF_NONE
```

Comment:
The name of the option contradicts its description: the option says that there's not default interface, while description says the first is default.